src/runtime/audience-action-local-flow-test.js

```diff
@@ -413,6 +413,41 @@ describes.realWin('AudienceActionLocalFlow', (env) => {
         expect(signinButton).to.be.null;
       });
 
+      it('escapes bad input', async () => {
+        const BAD_CONFIG = `
+          {
+            "publication": {
+              "name": "<script>PUBLICATOIN_NAME</script>"
```

Comment:
nit: PUBLICATION_NAME (my understanding is the html is what makes it a bad input and not the typo, correct me if i'm wrong)